src/main/drivers/accgyro/accgyro_mpu3050.c

```diff
@@ -48,6 +50,17 @@
 #define MPU3050_USER_RESET      0x01
 #define MPU3050_CLK_SEL_PLL_GX  0x01
 
+static uint8_t mpu3050GetDLPF(uint8_t lpf)
+{
+    uint8_t ret;
+    if (lpf == GYRO_HARDWARE_LPF_1KHZ_SAMPLE) {
+        ret = MPU3050_DLPF_188HZ;
```

Comment:
if we do the direct return here we can omit the `uint8_t ret` / single return. I'm not sure if it's actually better or more readable, though.

Reply:
This is an obsolete gyro anyway and no targets use it (so the code isn't even linked).  Just fixed it up because it wasn't difficult and it was really broken previously.  We'll probably remove this gyro at some point.

Reply:
Cool, that's also fine. :+1: